sys/aarch64/rootdev.c

```diff
@@ -166,7 +166,9 @@ static void bcm2835_intr_handle(bus_space_handle_t irq_base, bus_size_t offset,
 
   while (pending) {
     int irq = ffs(pending) - 1;
-    intr_event_run_handlers(events[irq]);
+    /* XXX: some pending bits are shared between BASIC and GPU0/1. */
+    if (events[irq])
```

Comment:
`intr_event_run_handlers` asserts that `ie != NULL`. If you get an interrupt and you don't have any associated
interrupt handler with it, you'll probably want to know it (and let it crush in the assert).

Reply:
Not in this case.
I really want to ignore duplicate interrupts here.